bluemira/geometry/parameterisations.py

```diff
@@ -986,6 +986,191 @@ def _make_control_points(p0, p3, theta0, theta3, l_start, l_end):
         return p1, p2
 
 
+class PictureFrameTools:
+    """
+    Curved Pictureframe parameterisation with straight (flat) inner limb
+
+    Parameters
+    ----------
+    var_dict: Optional[dict]
+        Dictionary with which to update the default values of the parameterisation.
+
+    """
+
+    @staticmethod
+    def _make_domed_leg(
```

Comment:
Also good to specify types in declaration (for all function declaration on this PR)

``` python
       axis,
        x_out: float,
        x_curve_start: float,
        x_mid: float,
        z_top: float,
        z_mid: float,
        flip=False,
        r_c=0,
    ):
```

Reply:
These seems to be `np.float` this syntax is then not valid

Reply:
using np.float directly is depreciated you should now just use float. (np.array if its an array)